config/index.js

```diff
@@ -654,6 +654,11 @@ var conf = convict({
       default: /.+@mozilla\.com$/,
       env: 'SIGNIN_CONFIRMATION_FORCE_EMAIL_REGEX'
     },
+    skipForEmailAddresses: {
```

Comment:
Can you explore what it'd be like to change this to an array and pass those values via an environment variable? If it's horrible, meh.

Reply:
Updated!